recipe/meta.yaml

```diff
@@ -1,33 +1,46 @@
 {% set version = "0.1.43" %}
 {% set name = "jaxlib" %}
 
+# compute right wheel filename for later
 {% set pyver = "cp{}{}".format(PY_VER[0], PY_VER[2]) %}
-
 {% set platform = "manylinux2010_x86_64" %}  # [linux]
 {% set platform = "macosx_10_9_x86_64" %}  # [osx]
-
 # {distribution}-{version}(-{build tag})?-{python tag}-{abi tag}-{platform tag}.whl.
 {% set fn = "{}-{}-{}-none-{}.whl".format(name, version, pyver, platform) %}
 
+# only use these for the URLS to help the bot
+# the urls below have to be explicit for the bot to work
+{% set linux_whl = "manylinux2010_x86_64" %}
+{% set osx_whl = "macosx_10_9_x86_64" %}
+
 package:
   name: jaxlib
   version: {{ version }}
 
 source:
-  fn: {{ fn }}
-  url: https://pypi.io/packages/{{ pyver }}/{{ name[0] }}/{{ name }}/{{ fn }}
-  sha256: 74490dd3c2e37492dbefc040a059bdedf0ba62d5d68c51d43181ac6c1aa14518  # [linux and py==36]
+  fn: wheel.whl
+
+  url: https://pypi.io/packages/cp36/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-cp36-none-{{ linux_whl }}.whl  # [linux and py==36]  
```

Comment:
@ericmjl this is quite ugly and redundant but it is the only way to make the new bot work here.

Maybe it is time to work on a source compilation again :smile:

Reply:
We might be able to improve the bot OFC, but no time for that right now.